src/Utils/ServiceYamlConfig.php

```diff
@@ -81,6 +82,11 @@ public function __construct(?string $serviceYaml)
             if (!is_null($apis)) {
                 $this->apiNames = Vector::new($apis)->map(fn ($a) => $a->getName());
             }
+
+            $publishing = $service->getPublishing();
+            if (!is_null($publishing)) {
```

Comment:
nit, but why not just
```suggestion
            if ($publishing = $service->getPublishing()) {
```

Reply:
Verified that the default value is `null`. Updated.